### ISSUE TITLE
CRITICAL: vmware photon service create does not honor flavor settings.

### ISSUE THREAD

**Issue by pompomJuice** [vmware/photon-controller#120]
Only this setting is honored:
`--vm_flavor value, -v value         VM flavor name for master and worker`

These two are completely ignored:
```
--master-vm-flavor value, -m value  Override master VM flavor
--worker-vm-flavor value, -W value  Override worker VM flavor
```

This is kind of a showstopper for me. Please help.

**Comment by pompomJuice** [comment]
Another way around this issue would have been to edit `service-other-vm` but something keeps spamming a new one all the time, one that only has 1 CPU and 4GB Ram.

**Comment by AlainRoy** [comment]
Yes, this is a bug in the CLI for Photon Controller 1.2. It has been fixed in 1.2.1, which will be released soon. I don't have an exact date for it, but i believe it's days, not weeks.

**Comment by pompomJuice** [comment]
That's awesome! 

Thanks.
